synthesizer/src/process/mod.rs

```diff
@@ -149,6 +149,61 @@ impl<N: Network> Process<N> {
         Ok(process)
     }
 
+    /// Initializes a new process.
+    #[inline]
+    pub fn load_transfer() -> Result<Self> {
+        //use web_sys::console;
+
+        let timer = timer!("Process::load");
+
+        // Initialize the process.
+        // let formatted_string = format!(" {:#?} : Loading Bytes", u_srs.);
+        // //console::log_1(&formatted_string.into());
+
+        let mut process = Self { universal_srs: Arc::new( UniversalSRS::load()?), stacks: IndexMap::new() };
+        // process.universal_srs().clone()..to_bytes();
+        lap!(timer, "Initialize process");
+
+        // Initialize the 'credits.aleo' program.
+        let program = Program::credits()?;
+        lap!(timer, "Load credits program");
+
+        // Compute the 'credits.aleo' program stack.
+        let stack = Stack::new(&process, &program)?;
+        lap!(timer, "Initialize stack");
+
+        // Synthesize the 'credits.aleo' circuit keys.
+        for function_name in program.functions().keys() {
+            if function_name.to_string() == "transfer" {
```

Comment:
I imagine we can expose interfaces here to support arbitrary programs & arbitrary functions on those programs. For most applications, users shouldn't need most prover files ie the credits mint file which is only used on the genesis block.